roles/setup-services/tasks/configure-keycloak.yml

```diff
@@ -24,6 +24,38 @@
          keycloak_auth_response.status in [200, 401, 403]
   ignore_errors: yes
 
+- name: Fetch public client JSON from keycloak
+  uri:
+    url: "https://{{ keycloak_route_output.stdout }}/auth/admin/realms/{{namespace}}/clients/{{PUBLIC_CLIENT_ID}}"
+    method: GET
+    validate_certs: no
+    body_format: json
+    headers:
+      Authorization: "Bearer {{ keycloak_auth_response.json.access_token }}"
+    status_code: [200]
+  register: public_client_original
+  ignore_errors: yes
+
+- name: Write keycloak client JSON to file
+  copy: content="{{public_client_original.json}}" dest=/tmp/keycloak-client-original.json
```

Comment:
Was unable to get `jq` and ansible to work nicely without writing this to disk. Happy to change if this is an issue and someone knows how to pass the `public_client_original.json` without getting garbled output from `jq`